jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -88,20 +142,26 @@ public static LoggingMXBeanImpl getInstance() {
 	@Override
 	public String getLoggerLevel(String loggerName) {
 		String result = null;
-		Logger logger = LogManager.getLogManager().getLogger(loggerName);
-
-		if (logger != null) {
-			// The named Logger exists. Now attempt to obtain its log level.
-			Level level = logger.getLevel();
-			if (level != null) {
-				result = level.getName();
-			} else {
-				// A null return from getLevel() means that the Logger
-				// is inheriting its log level from an ancestor. Return an
-				// empty string to the caller.
-				result = ""; //$NON-NLS-1$
+		
```

Comment:
Perhaps we should keep the old implementation for Java 8?

Reply:
It would be a little less risky to keep Java 8 unmodified, as there is some overhead that will occur when optimizing the reflection.

Reply:
Okay. I'll work on that.